apps/src/storage/dataBrowser/loadDataForView.js

```diff
@@ -0,0 +1,98 @@
+import {getStore} from '../../redux';
+import {DataView} from '../constants';
+import {
+  tableType,
+  addTableName,
+  deleteTableName,
+  updateTableColumns,
+  updateTableRecords,
+  updateKeyValueData,
+  updateTableList,
+} from '../redux/data';
+import {isDatablockStorage, isFirebaseStorage} from '../storage';
+
+let lastView;
+let lastTableName;
+let lastStorage;
+
+export function refreshCurrentDataView() {
+  // TODO: post-firebase-cleanup, remove if guard, but keep the contents: #56994
+  if (isDatablockStorage()) {
+    loadDataForView(lastStorage, lastView, null, lastTableName);
+  }
+}
```

Comment:
You'll see references to `refreshCurrentDataView()` throughout the code after UX actions, this is because in Firebase the callbacks resulted in reloading data after a state change, whereas in datablock storage, the reload must be manually kicked off.